README.md

```diff
@@ -21,11 +21,11 @@ QRLJacking attack consists of two sides:
 ### Our example will be: WhatsApp Web Application!
 
 ## Server Setup (Attacker's hosting):
-1. Upload "qrHandler.php" to your server, this php file is used to convert the base64 qr code string into a valid .JPG file
+1. Upload "qrHandler.php" to your server. This PHP file is used to convert the base64 QR code string into a valid .JPG file
 
-	Now we have a valid generated QR image named "tmp.jpg" residing in the same root folder of your files and will be updated whenever that php file will be called, So we can put it anywhere "for example a fake WhatsApp page, a scam page with an offer related to WhatsApp, etc... depending on your creativity"
+	Now we have a valid generated QR image named "tmp.jpg" residing in the same root folder of your files and will be updated whenever that php file will be called. We can put it anywhere, for example, on a fake WhatsApp page (a scam page with an offer related to WhatsApp). You can get creative with this! 
```

Comment:
"Now we have a valid generated QR image named "tmp.jpg" residing in the same root folder of your files and will be updated whenever that php file will be called." Forgot to capitalize PHP in that sentence.